contrib/CR3BP/CR3BP.py

```diff
@@ -0,0 +1,358 @@
+import numpy as np

```

Comment:
May you please provide a little description or links to the references you used when implementing the different equations within this script?